src/it/resources/data/DataBundleLogicIT.json

```diff
@@ -192,7 +192,33 @@
       "isPublishedEmailEnabled": true
     }
   },
-  "feedbackQuestions": {},
+  "feedbackQuestions": {
+    "qn1InSession1InCourse1": {
+      "id": "00000000-0000-4000-8000-000000000801",
+      "feedbackSession": {
+        "id": "00000000-0000-4000-8000-000000000701"
+      },
+      "questionDetails": {
+        "questionType": "TEXT",
+        "questionText": "What is the best selling point of your product?"
+      },
+      "description": "This is a text question.",
+      "questionNumber": 1,
+      "questionType": "TEXT",
```

Comment:
questionType seems to be repeated here. possible to remove this redundancy?

Reply:
Yes! Thanks for the catch!